### PR TITLE
Increase displayed properties, API version update.

### PR DESCRIPTION
**Pull request by kegor**
**Screenshots:**

image

### CODE REVIEW

File: src/Products/Metadata/Controllers/MetadataApiController.cs [hunk 1]
```diff
@@ -139,13 +136,20 @@ public HttpResponseMessage loadProperties(PostedDataEntity postedData)
...
                 else if (filePropertyCategory == FilePropertyCategory.Default)
                 {
-                    // Fetch all metadata properties that fall into a particular category
-                    IEnumerable<MetadataProperty> defaultProperties = metadata.FindProperties(p => p.Value.RawValue != null
-                                                                                                && !string.Empty.Equals(p.Value.RawValue)
-                                                                                                && (!p.Value.RawValue.GetType().IsValueType
-                                                                                                    || !p.Value.RawValue.Equals(Activator.CreateInstance(p.Value.RawValue.GetType())))
-                                                                                                && p.Tags.Any(t => t.Category == Tags.Content));
-
+                    IEnumerable<MetadataProperty> defaultProperties = new List<MetadataProperty>();
```
Comment (bobkovalex) [inline src/Products/Metadata/Controllers/MetadataApiController.cs:139]: Codacy found an issue: [Remove this useless assignment to local variable 'defaultProperties'.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-MVC/pullRequest?prid=5989453)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity decreasing per file
==============================
+ src/Products/Metadata/Controllers/MetadataApiController.cs  -4
         

Clones removed
==============
+ src/Products/Metadata/Controllers/MetadataApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-MVC/pullRequest?prid=5989453&bid=19640982)

File: src/Products/Metadata/Controllers/MetadataApiController.cs [hunk 2]
```diff
@@ -26,20 +26,21 @@ namespace GroupDocs.Metadata.MVC.Products.Metadata.Controllers
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class MetadataApiController : ApiController
     {
+        private static List<string> SupportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
```
Comment (bobkovalex) [inline src/Products/Metadata/Controllers/MetadataApiController.cs:29]: Codacy found an issue: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-MVC/pullRequest?prid=5989453)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/Products/Metadata/Controllers/MetadataApiController.cs  -4
         

Clones removed
==============
+ src/Products/Metadata/Controllers/MetadataApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-MVC/pullRequest?prid=5989453&bid=19640982)

**Review by Pavel-Teplitsky — APPROVED** [review]

**Pull request merged**